Gpt checker part 2 (#19)
greenelab/word_lapse_manuscript: content/04.results.md

## Models can be aligned and compared within and between years

We examined how the usage of tokens in biomedical text changes over time using machine learning models.
We trained the models to predict the actual token given a portion of its surrounding tokens, and each token was represented as a vector in a coordinate space constructed by the models.

> However, training these models is stochastic, resulting in arbitrary coordinate spaces.

Reviewer: ```suggestion

However, training these models is stochastic, resulting in arbitrary coordinate spaces.
Each model has its own unique coordinate space (Figure {@fig:word2vec_alignment}A), and each word is represented within that space (Figure {@fig:word2vec_alignment}B).
Model alignment is essential in allowing word2vec models to be compared [@doi:10.48550/arXiv.1605.09096; @doi:10.1038/s41597-021-01047-x].
Alignment projects every model onto a shared coordinate space (Figure {@fig:word2vec_alignment}C), enabling direct token comparison.
```

To enable comparison of the models, we aligned them onto a shared coordinate space.
We randomly selected 100 tokens to confirm that alignment worked as expected.

> We found that tokens in the global space were more similar to themselves within the year than between years, while identical tokens in unaligned models were completely distinct.

Reviewer: ```suggestion
We found that tokens in the global space were more similar to themselves within the year than between years, while identical tokens in unaligned models were completely distinct (Figure {@fig:word2vec_alignment}D).
```

> Local distances were unaffected by alignment, as token-neighbor distances remained unchanged.

Reviewer: ```suggestion
Local distances were unaffected by alignment, as token-neighbor distances remained unchanged (Figure {@fig:word2vec_alignment}D).
```

> Figures {@fig:word2vec_alignment}A-D show our results.

Reviewer: ```suggestion
```

We applied the CUSUM algorithm with integrated-model distance to correct for systematic differences in the underlying corpora.
We found 41281 terms with a detected changepoint from PMCOA and 2266 terms from preprints (Figures {@fig:preprint_published_changepoints}A and {@fig:preprint_published_changepoints}B).
Most of our detected changepoints (38019 for PMCOA and 2260 for preprints) only had a single event.

We detected a changepoint in PMCOA for 'cas9' from 2012 to 2013 (Figure {@fig:preprint_published_changepoints}C).
Before the changepoint, its closest neighbors were related to genetic elements (e.g., 'cas'1-3).
After the changepoint, its closest neighbors became terms related to targeting, sgRNA, gRNA, and other genome editing strategies, such as 'talen' and 'zfns' (Table {@tbl:cas9_neighbor_table}).
We detected change points for 'SARS' from 2002 to 2003 and 2019 to 2020 (Figure {@fig:preprint_published_changepoints}D), consistent with the emergences of SARS-CoV [@doi:10.1046/j.1440-1843.2003.00517.x] and SARS-CoV-2 [@doi:10.1016/j.ijantimicag.2020.105924; @doi:10.1038/s41564-020-0695-z] as observed human pathogens.

> Before each changepoint, the closest neighbors for 'SARS' were consideredmiscellaneous.

Reviewer: ```suggestion
Before each changepoint, the closest neighbors for 'SARS' were difficult to synthesize and summarize.
```

> After the even occurs, the neighbors for 'SARS' were consistent with the acronym for Severe Acute Respiratory Syndrome (Tables {@tbl:sars_neighbor_table_one} and {@tbl:sars_neighbor_table_two}).

Reviewer: ```suggestion
After changepoints, the neighbors for 'SARS' were consistent with the acronym for Severe Acute Respiratory Syndrome (Tables {@tbl:sars_neighbor_table_one} and {@tbl:sars_neighbor_table_two}).
```
